ui/com/user-info.jsx

```diff
@@ -25,8 +25,8 @@ const FLAG_DROPDOWN = [
 class AutoRefreshingComponent extends React.Component {
   constructor(props) {
     super(props)
-    this.state = this.computeState()
-    this.refreshState = props => { this.setState(this.computeState(props)) }
+    this.computeState(null, this.setStateCallback.bind(this))
```

Comment:
Oh-- seeing that you have to use the CB twice, I'd just put it within the function. Just seems weird to be a class method to me.

```js
const handleCB = (err, state) => {
  if (err) throw err
  this.setState(state)
}
this.computeState(null, handleCB)